wandb/sdk/data_types/base_types/media.py

```diff
@@ -263,6 +265,10 @@ def __eq__(self, other: object) -> bool:
             and self._sha256 == other._sha256
         )
 
+    @staticmethod
```

Comment:
I have it as a static method since self._path isn't what is always being checked so self isn't necessary, but looking through the code if i refactor some of the old code it could be made into a module scoped function. probably just going to leave it as is for now though